src/main/java/com/redhat/cloud/custompolicies/app/model/engine/FullTrigger.java

```diff
@@ -39,6 +41,9 @@ public FullTrigger() {
     trigger = new Trigger();
     conditions = new ArrayList<>();
     trigger.actions = new HashSet<>();
+    actionToTriggerActionMap = new HashMap<>();
+    actionToTriggerActionMap.put("webhook","hooks");
+    actionToTriggerActionMap.put("email","email");
```

Comment:
One last request, could you please use this `actionToTrigger` map for `storeActions` method ? I see it is still using the old if-else.

Reply:
Isn't that "webhook" -> "hooks" dangerous? What happens when we start to use "webhook" (which will require parameters and is not compatible with the old one) ? Will it need "realwebhookpleasestandup", "webhook" ?

Reply:
Good idea.